# Two-Tailed Test of Population Mean with Known Variance

### <font color='green'>Problem</font>

Suppose the mean weight of King Penguins found in an Antarctic colony last year was 15.4 kg. In a sample of 35 penguins same time this year in the same colony, the mean penguin weight is 14.6 kg. Assume the population standard deviation is 2.5 kg. At .05 significance level, can we reject the null hypothesis that the mean penguin weight does not differ from last year?

### <font color='blue'>Solution 1</font>

H0:μ = 15.4.

Ha:μ ≠ 15.4.


In [4]:
xbar = 14.6            # sample mean
mu0 = 15.4             # hypothesized value
sigma = 2.5            # population standard deviation
n = 35                 # sample size
z = (xbar-mu0)/(sigma/sqrt(n))
                      # test statistic
z

[1] -1.893146

In [5]:
# compute the critical value at .05 significance level.
alpha = .05
z.half.alpha = qnorm(1-alpha/2) # returns the associated Z-score"
c(-z.half.alpha, z.half.alpha)

[1] -1.959964  1.959964

**The test statistic -1.8931 lies between the critical values -1.9600 and 1.9600. Hence, at .05 significance level, we do not reject the null hypothesis that the mean penguin weight does not differ from last year.**

 ### <font color='blue'>Solution 2</font>

In [6]:
pval = 2 * pnorm (z)  # It doubles the lower tail p-value as the sample mean is less than the hypothesized value
pval                  # two−tailed p−value


[1] 0.05833852

**it is greater than the .05 significance level, we do not reject the null hypothesis**

# Two-Tailed Test of Population Mean with Unknown Variance

### <font color='green'>Problem</font>

Suppose the mean weight of King Penguins found in an Antarctic colony last year was 15.4 kg. In a sample of 35 penguins same time this year in the same colony, the mean penguin weight is 14.6 kg. Assume the sample standard deviation is 2.5 kg. At .05 significance level, can we reject the null hypothesis that the mean penguin weight does not differ from last year?

### <font color='blue'>Solution 1</font>

H0:μ = 15.4.

Ha:μ ≠ 15.4.

In [7]:
xbar = 14.6            # sample mean 
mu0 = 15.4             # hypothesized value 
s = 2.5                # sample standard deviation 
n = 35                 # sample size 
t = (xbar-mu0)/(s/sqrt(n))
t                      # test statistic 

[1] -1.893146

In [9]:
# compute the critical values at .05 significance level.
alpha = .05 
t.half.alpha = qt(1-alpha/2, df=n-1) 
c(-t.half.alpha, t.half.alpha) 

[1] -2.032245  2.032245

**The test statistic -1.8931 lies between the critical values -2.0322, and 2.0322. Hence, at .05 significance level, we do not reject the null hypothesis that the mean penguin weight does not differ from last year**

### <font color='blue'>Solution 2</font>

In [10]:
# compute the two-tailed p-value of the test statistic
pval = 2* pt(t, df=n-1)  # It doubles the lower tail p-value as the sample mean is less than the hypothesized value.
pval                      # two−tailed p−value 

[1] 0.06687552

**it is greater than the .05 significance level, we do not reject the null hypothesis that μ = 15.4**

# Lower (left) Tail Test of Population Mean with Known Variance

### <font color='green'>Problem</font>

Suppose the manufacturer claims that the mean lifetime of a light bulb is more than 10,000 hours. In a sample of 30 light bulbs, it was found that they only last 9,900 hours on average. Assume the population standard deviation is 120 hours. At .05 significance level, can we reject the claim by the manufacturer?

### <font color='blue'>Solution 1</font>

H0: μ ≥ 10000

Ha: μ < 10000

In [11]:
xbar = 9900            # sample mean 
mu0 = 10000            # hypothesized value 
sigma = 120            # population standard deviation 
n = 30                 # sample size 
z = (xbar-mu0)/(sigma/sqrt(n))
z                      # test statistic

[1] -4.564355

In [2]:
#  compute the critical value at .05 significance level.
alpha = .05
z.alpha = qnorm(1-alpha)
-z.alpha               # critical value

[1] -1.644854

**The test statistic -4.5644 is less than the critical value of -1.6449. Hence, at .05 significance level, we reject the claim that mean lifetime of a light bulb is above 10,000 hours.**

### <font color='blue'>Solution 2</font>

In [12]:
# compute the lower tail p-value
pval = pnorm(z)
pval                   # lower tail p−value 

[1] 2.505166e-06

**less than the .05 significance level, we reject the null hypothesis**

# Lower (left) Tail Test of Population Mean with Unknown Variance

### <font color='green'>Problem</font>

Suppose the manufacturer claims that the mean lifetime of a light bulb is more than 10,000 hours. In a sample of 30 light bulbs, it was found that they only last 9,900 hours on average. Assume the sample standard deviation is 125 hours. At .05 significance level, can we reject the claim by the manufacturer?

### <font color='blue'>Solution 1</font>

H0: μ ≥ 10000

Ha: μ < 10000

In [13]:
xbar = 9900            # sample mean 
mu0 = 10000            # hypothesized value 
s = 125                # sample standard deviation 
n = 30                 # sample size 
t = (xbar-mu0)/(s/sqrt(n)) 
t                      # test statistic 

[1] -4.38178

In [14]:
# compute the critical value at .05 significance level.
alpha = .05 
t.alpha = qt(1-alpha, df=n-1) 
-t.alpha               # critical value 


[1] -1.699127

**The test statistic -4.3818 is less than the critical value of -1.6991. Hence, at .05 significance level, we can reject the claim that mean lifetime of a light bulb is above 10,000 hours**

### <font color='blue'>Solution 2</font>

In [15]:
# compute the lower tail p-value of the test statistic
pval = pt(t, df=n-1) 
pval                   # lower tail p−value

[1] 7.035026e-05

**it is less than the .05 significance level, we reject the null hypothesis that μ ≥ 10000.**

# Upper (right) Tail Test of Population Mean with Known Variance

### <font color='green'>Problem</font>

Suppose the food label on a cookie bag states that there is at most 2 grams of saturated fat in a single cookie. In a sample of 35 cookies, it is found that the mean amount of saturated fat per cookie is 2.1 grams. Assume that the population standard deviation is 0.25 grams. At .05 significance level, can we reject the claim on food label?

### <font color='blue'>Solution 1</font>

H0: μ ≤ 2

Ha: μ > 2

In [16]:
xbar = 2.1             # sample mean 
mu0 = 2                # hypothesized value 
sigma = 0.25           # population standard deviation
n = 35                 # sample size 
z = (xbar-mu0)/(sigma/sqrt(n))
z                      # test statistic


[1] 2.366432

In [17]:
# compute the critical value at .05 significance level.
alpha = .05
z.alpha = qnorm(1-alpha) 
z.alpha                # critical value 

[1] 1.644854

**The test statistic 2.3664 is greater than the critical value of 1.6449. Hence, at .05 significance level, we reject the claim that there is at most 2 grams of saturated fat in a cookie**

### <font color='blue'>Solution 2</font>

In [6]:
# compute the upper tail p-value
pval = pnorm(z, lower.tail=FALSE) 
pval                   # upper tail p−value 

[1] 0.008980239

**less than the .05 significance level, we reject the null hypothesis**

In [20]:
# compute the lower tail p-value of the test statistic
pval = pt(t, df=n-1)
pval                   # lower tail p−value 

[1] 0.03343776

**it is ess than the .05 significance level, we reject the null hypothesis that μ ≥ 10000.**

# Upper (right) Tail Test of Population Mean with Unknown Variance

### <font color='green'>Problem</font>

Suppose the food label on a cookie bag states that there is at most 2 grams of saturated fat in a single cookie. In a sample of 35 cookies, it is found that the mean amount of saturated fat per cookie is 2.1 grams. Assume that the sample standard deviation is 0.3 gram. At .05 significance level, can we reject the claim on food label?

### <font color='blue'>Solution 1</font>

H0: μ ≤ 2

Ha: μ > 2

In [19]:
xbar = 2.1             # sample mean 
mu0 = 2                # hypothesized value 
s = 0.3                # sample standard deviation 
n = 35                 # sample size 
t = (xbar-mu0)/(s/sqrt(n)) 
t                      # test statistic 

[1] 1.972027

In [20]:
# compute the critical value at .05 significance level.
alpha = .05 
t.alpha = qt(1-alpha, df=n-1) 
t.alpha                # critical value

[1] 1.690924

**The test statistic 1.9720 is greater than the critical value of 1.6991. Hence, at .05 significance level, we can reject the claim that there is at most 2 grams of saturated fat in a cookie**

### <font color='blue'>Solution 2</font>

In [21]:
# compute the upper tail p-value of the test statistic
pval = pt(t, df=n-1, lower.tail=FALSE) 
pval                   # upper tail p−value 

[1] 0.02839295

**it is ess than the .05 significance level, we reject the null hypothesis that μ ≤ 2**


### <font color='red'>Remember</font> 

**known Variance**<br>
qnorm ()  What is the Z-score of the -th quantile of the normal distribution<br> pnorm ()  calculates cumulative distribution function of normal distribution

**Unknown Variance**<br>
qt () returns the value of the inverse cumulative density function of the Student t distribution given a certain random variable x and degrees of freedom df.<br>
pt () returns the value of the cumulative density function  of the Student t distribution given a certain random variable x and degrees of freedom df

**with upper tail**<br>
add this argument <font color='red'>lower.tail=FALSE</font>  to pnorm() and pt()
